Source/Wit/Private/Voice/Capture/Emulation/VoiceCaptureEmulation.cpp

```diff
@@ -232,10 +272,16 @@ bool FVoiceCaptureEmulation::Tick(float DeltaTime)
 		if (bIsAnyDataToCopy)
 		{
 			UncompressedAudioBuffer.AddUninitialized(DataSizeToCopy);
-		
-			const uint8* SoundData = static_cast<const uint8*>(SoundWave->RawData.LockReadOnly());
-			FMemory::Memcpy(UncompressedAudioBuffer.GetData(), &SoundData[LastDataIndex], DataSizeToCopy);
-			SoundWave->RawData.Unlock();
+			if (bIsPackagedBuild)
+            {
+				FMemory::Memcpy(UncompressedAudioBuffer.GetData(), &DecompressedRawPCMData[LastDataIndex], DataSizeToCopy);
```

Comment:
Just wondering we are accessing DecompressedRawPCMData using LastDataIndex which is calculated using ElementCount. ElementCount comes from SoundWave->RawPCMDataSize but earlier we size the array using a different value of SoundQualityInfo.SampleDataSize. Are we sure these two values match and that we aren't potentially indexing out of bounds?

Reply:
good catch. yes, they match. to avoid confusing,  i will change the code to not mixing.   will just use one of them.